JamGame/Assets/Scripts/Level/DragAndDropManager.cs

```diff
@@ -26,19 +26,13 @@ public class DragAndDropManager : MonoBehaviour
         private CoreModel bufferCoreModel;
 
         [SerializeField]
-        [Pickle(typeof(IDragAndDropAgent), LookupType = ObjectProviderType.Scene)]
-        private GameObject backupDragAndDropProvider;
-        private IDragAndDropAgent backupDragAndDrop;
+        private SerializableInterface<IDragAndDropAgent> backupDragAndDrop;
```

Comment:
Revert to pickle here as it's used everywhere else

Reply:
Pickle gives a better object picker from assets and works wired with interface.
This `SerializableInterface` gives an easy way to work with interfaces.

If it needed, i can fork all git repos to our company for controlling source of all project dependencies.

Reply:
> If it needed

No, it's not needed, just don't need to pull extra dependency to use just in one place, especially when you already have working solution